src/game/models/equips.js

```diff
@@ -310,7 +310,7 @@ export default [
   {
     id: 'ice_ring' ,
     name: 'Ice Ring',
-    description: 'It looks as powerfull as it shines.',
+    description: 'Ice rink?! Imagine how cray, da &^@#ing szie fo dose s#!%!!!1!',
```

Comment:
I fixed the typo, and this joke is about the "narrator" understanding wrong what he was supposed to describe, but given it had to be explained, I imagine it should be changed?